### PR TITLE
Mitos2: mising genes output

### PR DESCRIPTION
**Pull request by gallardoalba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [x] - This PR does something else (explain below)

This PR adds an additional output and the corresponding test.

### CODE REVIEW

File: tools/mitos/mitos2.xml [hunk 1]
```diff
@@ -52,6 +52,10 @@ $advanced_ncrna.sensitive
   --noplots
 #end if
 
+#if "missing" in str($addoutputs).split(','):
```
Comment (bgruening) [inline tools/mitos/mitos2.xml:55]: I guess you do not need to split you can do a string search instead.

Comment (gallardoalba) [reply tools/mitos/mitos2.xml:55]: Yes, I know, but it is how the rest of if conditionals have been written, so just decided to maintain the style. I'll change it.

Comment (bgruening) [reply tools/mitos/mitos2.xml:55]: All good then, this was just a small hint, not important.

**Comment by bgruening** [comment]
> .. WARNING: Missing shed metadata field [long_description] for repository

This lint could be fixed

File: tools/mitos/mitos2.xml (hunk 1 above)
Comment (gallardoalba) [reply tools/mitos/mitos2.xml]: Fixed I think.

**Review by bernt-matthias — APPROVED** [review]

**Pull request merged**